themes/salted/index.html

```diff
@@ -325,34 +329,36 @@
-                                            <tr>
-                                                <td align="center" style="padding: 0 0 0 0; font-size: 16px; line-height: 25px; font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; color: #666666;" class="padding-copy">
-                                                    <p><%- action.instructions %></p>
-                                                </td>
-                                            </tr>
-                                        </table>
-                                    </td>
-                                </tr>
-                                <tr>
-                                    <td>
-                                        <!-- BULLETPROOF BUTTON -->
-                                        <table align="center" width="100%" border="0" cellspacing="0" cellpadding="0" class="mobile-button-container">
-                                            <tr>
-                                                <td align="center" style="padding: 25px 0 0 0;" class="padding-copy">
-                                                    <table border="0" cellspacing="0" cellpadding="0" class="responsive-table">
-                                                        <tr>
-                                                            <td align="center"><a href="<%- action.button.link %>" target="_blank" style="font-size: 16px; font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; font-weight: normal; color: #ffffff; text-decoration: none; background-color: <%- action.button.color %>; border-top: 15px solid <%- action.button.color %>; border-bottom: 15px solid <%- action.button.color %>; border-left: 25px solid <%- action.button.color %>; border-right: 25px solid <%- action.button.color %>; border-radius: 3px; -webkit-border-radius: 3px; -moz-border-radius: 3px; display: inline-block;" class="mobile-button"><%- action.button.text %> &rarr;</a></td>
-                                                        </tr>
-                                                    </table>
-                                                </td>
-                                            </tr>
-                                        </table>
-                                    </td>
-                                </tr>
+                            <% action.forEach(function (actionItem) { -%>
+                                    <tr>
+                                        <td>
+                                            <!-- COPY -->
+                                            <table align="center" width="100%" border="0" cellspacing="0" cellpadding="0">
+                                                <tr>
+                                                    <td align="center" style="padding: 0 0 0 0; font-size: 16px; line-height: 25px; font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; color: #666666;" class="padding-copy">

```

Comment:
A little visual "bug" in the salted theme:

![screen shot 2016-10-01 at 11 30 20 pm](https://cloud.githubusercontent.com/assets/4394189/19016861/4a297fcc-882f-11e6-96fc-a36f3269c222.png)

There is insufficient padding between the action items.

We should change the `padding` declaration in line 338 and specify `25px` for the top, to avoid this situation.

We should then remove `25px` padding from the parent `<td>` that contains the action buttons so that the padding-top isn't too high, in line 325.

The new padding for the parent `<td>` (L325):
```
padding: 15px 15px 40px 15px;
```

Which gives us:
![screen shot 2016-10-01 at 11 35 27 pm](https://cloud.githubusercontent.com/assets/4394189/19016885/c1dfa87a-882f-11e6-8ed7-87bd72e5aa99.png)